### ISSUE TITLE
Feature: Ignore partial support

### ISSUE THREAD

**Issue by ncoden** [anandthakker/doiuse#55]
In my use of `doiuse` for [Foundation](https://github.com/zurb/foundation-sites), I have a lot of properties that are _partially_ supported.

The most common are:
- `rem` and `transition` by IE9/10
- `box-shadow` and `background-image` options by Android Browser 2.3/3

Currently, there is so much _partial support_ errors that the _most important_ errors can be easily forgotten.

I don't need to have a full support of these old browsers, so the errors that must be shown are only the unsupported properties for these browsers, and the unsupported or partially supported properties for the newer browsers. I think it would be very useful to be able to ignore all partial support errors, or for a specific list of browsers.

For example, with gulp:
```node
var gulp = require('gulp');
var postcss = require('gulp-postcss');
var doiuse = require('doiuse');

var FULL_COMPATIBILITY = [
  'last 2 versions',
  'ie >= 10',
  'Android >= 4',
  'ios >= 7'
];

var PARTIAL_COMPATIBILITY = [
  'ie >= 9',
  'Android >= 2.3',
];

gulp.task('doiuse', function() {
  return gulp.src(['_build/assets/css/foundation.css'])
    .pipe(postcss([doiuse({
      fullCompatibilityBrowsers: FULL_COMPATIBILITY,
      partialCompatibilityBrowsers: PARTIAL_COMPATIBILITY,
      onFeatureUsage: function (usageInfo) {
        console.log(usageInfo.message)
      }
    })]))
});
```

What do you think ?

**Comment by anandthakker** [comment]
@ncoden Hmm.  If I'm not mistaken, this is something you could already do by looking at `usageInfo.featureData.missing` and `usageInfo.featureData.partial` in your `onFeatureUsage` function.  Is that workable for you?

**Comment by Nemo64** [comment]
I there a concrete use case for this? I think of the `css-gradients` warning (that I actually caused but meh...) but I wouldn't remove iOS and Safari to my partial support list. In that case I'd much rather ignore `css-gradients` in those specific browsers because I either know the risk or have the appropriate [postcss plugin installed](https://github.com/ismay/stylelint-no-unsupported-browser-features).

**Comment by kristerkari** [comment]
> I there a concrete use case for this?

Yes there is. The stylelint plugin `stylelint-no-unsupported-browser-features` would need a new option that would ignore partially supported features. 

In my project I'm only interested in knowing which properties are not supported at all in the browsers that I have defined in my browserslist. I want to ignore the partially supported features because in most cases those are features that work just fine, but might have some limitations.

With the way that `doiuse` currently works, you would have to first run `doiuse` to get a list of partially supported features, and then run `doiuse` again to ignore those partially supported features.

What would be needed is some flag in the result that tells if the feature is partially supported, instead of having that info in the `onFeatureUsage` callback.

**Comment by Nemo64** [comment]
> only interested in knowing which properties are not supported at all in the browsers 

And that is the difficult part. The css features aren't specific enough to tell you that. If you use css-grids (which is partially supported in IE 11) you might not notice that it is completely broken. And if you say that "you should know" than why use doiuse at all?

**Comment by kristerkari** [comment]
> And if you say that "you should know" than why use doiuse at all?

As I said I'm only interested in seeing which CSS features are not supported at all. I use MDN or caniuse.com if I want to see which browser limitations certain CSS features have.

**Comment by eduardomoroni** [comment]
I ran into similar issue, partial support is optional but full support is mandatory. I am using an ESLint to do the check on the pipeline.

https://gitlab.com/ismay/stylelint-no-unsupported-browser-features/issues/87 for reference
